webpack.config.babel.js

```diff
@@ -150,8 +151,5 @@ export default env => {
     resolve: {
       alias: extraResolveAliases,
     },
-    output: {
-      jsonpFunction: 'coreJsonp',
-    },
```

Comment:
I think removing this may break plugins; those are separate webpack builds and thus need separate callbacks.

Reply:
What was the cause for adding it?

Reply:
@pferreir may remember the details, otherwise the git blame and then the commit/PR adding it may also help..

Reply:
https://github.com/indico/indico/commit/902c217299d89c927ac9df961bff4c1946dfc623

Reply:
> This keeps the plugin environment from colliding with the core in
production builds.

Reply:
https://webpack.js.org/blog/2020-10-10-webpack-5-release/#automatic-unique-naming

So I guess this is no longer needed :)